.github/workflows/pr.yml

```diff
@@ -125,14 +129,12 @@ jobs:
   build-android-ios-065:
     uses: ./.github/workflows/ios_android.yml
     with:
-      java-version: '8'
```

Comment:
The VM images already have Java 8, 11, 17 installed. We don't need the extra steps to install JDK manually since we can just point the JAVA_HOME environment variable to the installed one.